### PR TITLE
Fix generate Format tag

### PR DESCRIPTION
**Pull request by tltk90**
Could solve #962

### CODE REVIEW

**Comment by KaiVolland** [comment]
Hey @tltk90, thanks for your contribution.

Could you add a new test with a corresponding SLD and geostyler-style?

File: src/SldStyleParser.ts [hunk 1]
```diff
@@ -1901,13 +1901,19 @@ export class SldStyleParser implements StyleParser<string> {
         case 'png':
         case 'jpeg':
         case 'gif':
-          graphic[0][ExternalGraphic][0][Format] = [`image/${iconExt}`];
+          graphic[0][ExternalGraphic][1][Format] = [{
```
[the remark this answers is not in the document]
Comment (tltk90) [reply src/SldStyleParser.ts:1904]: Yes. if it is on `[ExternalGraphic][0]` the `Format` was ignore.

**Comment by tltk90** [comment]
@KaiVolland I think the existing tests are sufficient, I had a spelling error, that's why the tests failed earlier.
I fixed it. The test `can write a SLD PointSymbolizer with ExternalGraphic`

**Comment by KaiVolland** [comment]
If the tests were sufficient they should have failed before your MR :smile: . Or in other words; could you add a test that demonstrates the correct translation of a graphic with an extension?

**Comment by tltk90** [comment]
Sorry on the delay with the answer,
The "problem" with the tests is the `Format` tag is not part of the geostyler object,
and in all the tests we you need to compare sld you convert it back to geostyler object,
so you don't catch this problem.

You can see this in the test 
`can write a SLD 1.1 PointSymbolizer with ExternalGraphic svg`
if you print the `sldString` before convert it back to geostyler you see the format was missing.

**Comment by KaiVolland** [comment]
Thanks for the contribution @tltk90 :pray:

**Pull request merged**